solutions/cf-1560B.mdx

```diff
@@ -0,0 +1,133 @@
+---
+id: cf-1560B
+source: CF
+title: Who's Opposite?
+author: Ryan Chou, Nathan Gong
+---
+
+[Official Editorial (C++)](https://codeforces.com/blog/entry/94009)
+
+## Explanation
+
+Since the number of people left of $a$ and the number of people right of $b$ must be equal, we can find the total number of people in the circle with $|a - b| \cdot 2$. 
+
+From here, we can see if this circle is possible with $a, b, c > \texttt{numGroup}$, 
+
+If any of these exceed the number of people in the circle, they can't have a partner. 
+
+Otherwise, we can subtract or add $|a - b|$ from $c$ to find the answer.
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(1)$
+
+<LanguageSection>
+
+<CPPSection>
+
+```cpp
+#include <iostream>
+#include <stdlib.h>
+using namespace std;
+
+int solve(long a, long b, long c) {
+	long numGroup = abs(a - b) * 2;
```

Comment:
`long` in c++ is not necessarily larger than `int`, you probably want `long long`

Reply:
i thought i changed this

Reply:
oops